board/stm32fx/llrtc.h

```diff
@@ -1,6 +1,8 @@
 #define RCC_BDCR_MASK_LSE (RCC_BDCR_RTCEN | RCC_BDCR_RTCSEL | RCC_BDCR_LSEMOD | RCC_BDCR_LSEBYP | RCC_BDCR_LSEON)
 #define RCC_BDCR_MASK_LSI (RCC_BDCR_RTCEN | RCC_BDCR_RTCSEL)
 
+#define RCC_BDCR_LSEDRV_1 0U
```

Comment:
this is pretty hacky, we should just do a better job of splitting up the H7 and F4 drivers

Reply:
yeah that's the reason it's still in draft, needs some cleanup